scilpy/utils/metrics_tools.py

```diff
@@ -212,3 +212,30 @@ def plot_metrics_stats(mean, std, title=None, xlabel=None,
 
     plt.close(fig)
     return fig
+
+
+def get_metrics_stats_over_volume(weights, metrics_files):
```

Comment:
@EmmaRenauld What was the nomenclature you used to replace <over_streamlines> since it was not clear ?
The same should be used here

Reply:
I used along to mean point per point.

For "over" meaning the whole bundle, I just removed over. 
get_metrics_stats_over_streamlines became get_bundle_metrics_mean_std and in the description I said "Returns the mean value of each metric for the whole bundle"

Reply:
so... Get_volume_metrics_stats?

Reply:
get_roi_metrics_stats final answer !